main.css

```
@@ -1,5 +1,5 @@
 .head {
-    background: #fc0;
+    background: red;
```

Marina, ¿estás segura de que el color de fondo era rojo? Me suena que era este tono, ¿no? Grs
```suggestion
    background: #567808;
```

---

```suggestion
    background: purple;
```

---

Guay, malditos diseñadores. Lo commiteo!